src/spesh/inline.c

```diff
@@ -270,14 +270,18 @@ MVMSpeshGraph * MVM_spesh_inline_try_get_graph(MVMThreadContext *tc, MVMSpeshGra
 
     /* Build graph from the already-specialized bytecode and check if we can
      * inline the graph. */
-    ig = MVM_spesh_graph_create_from_cand(tc, target_sf, cand, 0, &deopt_usage_ins);
+    MVMROOT(tc, cand, {
+        ig = MVM_spesh_graph_create_from_cand(tc, target_sf, cand, 0, &deopt_usage_ins);
+    });
```

Comment:
I think this doesn't allocate.

Reply:
Not sure why I added it, removing it doesn't cause check-roots to complain.